en/guide/routing.md

```diff
@@ -68,19 +68,24 @@ app.all('/secret', (req, res, next) => {
 
 Route paths, in combination with a request method, define the endpoints at which requests can be made. Route paths can be strings, string patterns, or regular expressions.
 
-The characters `?`, `+`, `*`, and `()` are subsets of their regular expression counterparts. The hyphen (`-`) and the dot (`.`) are interpreted literally by string-based paths.
+{% capture caution-character %} In express 5, the characters `?`, `+`, `*`, `[]`, and `()` are handled differently than in version 4, please review the [migration guide](/{{ page.lang }}/guide/migrating-5.html#path-syntax) for more information.{% endcapture %}
 
-If you need to use the dollar character (`$`) in a path string, enclose it escaped within `([` and `])`. For example, the path string for requests at "`/data/$book`", would be "`/data/([\$])book`".
+{% include admonitions/caution.html content=caution-character %}
 
-<div class="doc-box doc-info" markdown="1">
+{% capture note-dollar-character %}In express 5, you don't need to escape the dollar character (`$`) with `\`.
+{% endcapture %}
+
+{% include admonitions/note.html content=note-dollar-character %}
+
+{% capture note-path-to-regexp %}
   Express uses [path-to-regexp](https://www.npmjs.com/package/path-to-regexp) for matching the route paths; see the path-to-regexp documentation for all the possibilities in defining route paths. [Express Route Tester](http://forbeslindesay.github.io/express-route-tester/) is a handy tool for testing basic Express routes, although it does not support pattern matching.
```

Comment:
Although not part of this edit, I'd probably remove the link to the route tester or open an issue to try and get it updated to the version used by Express 5. We could also fork it and update it as part of the website potentially.

Reply:
It's true, it's not updated. I will remove it in the meantime. I would opt to fork the repository so we're not dependent on the author. Maybe we can ask him to transfer it to the organization and we can maintain it